econml/metalearners.py

```diff
@@ -141,17 +147,19 @@ def fit(self, Y, T, X, inference=None):
 
         Returns
         -------
-        self: an instance of self.
+        self : an instance of self.
         """
         # Check inputs
         Y, T, X, _ = check_inputs(Y, T, X, multi_output_T=False)
-        if not np.array_equal(np.unique(T), [0, 1]):
-            raise ValueError("The treatments array (T) can only contain 0 and 1.")
-        feat_arr = np.concatenate((X, T.reshape(-1, 1)), axis=1)
+        T = self._label_encoder.fit_transform(T)
+        T = self._one_hot_encoder.fit_transform(T.reshape(-1, 1))
+        self._d_t = T.shape[1:]
```

Comment:
Again I would expect this to be `(T.shape[1] - 1,)` for consistency with the rest of the library.

Reply:
(and for T to omit its first column)